cmd/direct-csi/cmd.go

```diff
@@ -25,7 +25,7 @@ import (
 	"github.com/spf13/cobra"
 	"github.com/spf13/viper"
 
-	_ "github.com/golang/glog"
+	_ "k8s.io/klog"
```

Comment:
if you set --v, it wont understand the argument. You need to initialize flags like its done here: https://github.com/kubernetes-sigs/container-object-storage-interface-controller/blob/master/cmd/controller-manager/controller-manager.go#L37

Reply:
Done